spec/spec_helper.sh

```
@@ -0,0 +1,34 @@
+# spec/spec_helper.sh
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)